noah/src/main/scala/edu/uci/ics/cloudberry/noah/kafka/AsterixDataInsertion.scala

```diff
@@ -1,12 +1,16 @@
-package edu.uci.ics.cloudberry.noah.feed
+package edu.uci.ics.cloudberry.noah.kafka
 
 import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
+import edu.uci.ics.cloudberry.noah.feed._
+import org.apache.kafka.clients.consumer.ConsumerRecords
 import play.api.libs.ws.ahc.AhcWSClient
 
+import scala.collection.JavaConversions._
 import scala.concurrent.ExecutionContext.Implicits.global
 
-object AsterixHttpRequest {
+
+class AsterixDataInsertion {
 
   def createClient(): AhcWSClient = {
```

Comment:
Moved the wsClient creation to GeneralTopicConsumer and put it as a parameter for asterixdatainsertoin. Check if it is better